### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by miiu96**
## Reasoning behind the pull request
- Changed the URL from the `config.toml` for the stable tag location

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+[![API Reference](https://godoc.org/github.com/multiversx/mx-chain-go?status.svg)](https://godoc.org/github.com/multiversx/mx-chain-go)
+[![codecov](https://codecov.io/gh/multiversx/mx-chain-go/branch/master/graph/badge.svg?token=MYS5EDASOJ)](https://codecov.io/gh/multiversx/mx-chain-go)
 
-# Elrond go
+# mx-chain-go
 
-The go implementation for the Elrond Network protocol
+The go implementation for the MultiversX protocol
 
 ## Installation and running
```
Comment (bogdan-rosianu) [inline README.md:20]: remove docker reference on L22

**Review by iulianpascalau — COMMENTED** [review]
only what Bogdan found

File: README.md (hunk 1 above)
Comment (miiu96) [reply README.md:20]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -30,26 +29,26 @@ The main branch that will be used is the master branch. Alternatively, an older
...
+$ cd $GOPATH/src/github.com/multiversx/mx-chain-go/cmd/keygenerator
 $ go build
 $ ./keygenerator
 ```
 
 ### Start the node 
-#### Step 4a: Join Elrond testnet:
-Follow the steps outlined [here](https://docs.elrond.com/validators/elrond-go-scripts/config-scripts/). This is because in order to join the testnet you need a specific node configuration.
+#### Step 4a: Join MultiversX testnet:
+Follow the steps outlined [here](https://docs.multiversx.com/validators/nodes-scripts/config-scripts/). This is because in order to join the testnet you need a specific node configuration.
```
Comment (raduchis) [inline README.md:51]: this is not live yet? i get a "restricted" message

**Review by raduchis — APPROVED** [review]

**Pull request merged**